routes/auth.js

```diff
@@ -7,10 +7,22 @@ const jwt = require('jsonwebtoken');
 const jwtDecode = require('jwt-decode');
 
 
-router.get('/users', passport.authenticate('jwt', { session: false }), async (req, res) => {
+router.get('/users', async (req, res) => {
   try {
     const users = await User.find()
+    // console.log(users)
     res.json(users)
+
+  } catch (e) {
+    res.status(500).json({ 'message': e.message });
+  }
+})
+
+router.post('/getOneUser', async (req, res) => {
+  try {
+    const user = await User.find({ email: req.body.email })
```

Comment:
There is a chance that the user object doesn't exist. I'm happy leaving this code how it is, but its something to be aware of on the client side

Reply:
I dont think this call will find a not exist object, since it call by clicking detail button on the user list.
Every user and its email must be exist, so that it can show in the list